### PR TITLE
[RHODS-7871] ODS-395 Verify App name

### PR DESCRIPTION
**Pull request by nehachopra27**
Automate https://polarion.engineering.redhat.com/polarion/#/project/OpenDataHub/workitem?id=ODS-395

### CODE REVIEW

File: ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot [hunk 1]
```diff
@@ -62,6 +62,15 @@ Verify Filters Are Working On Resources Page
     Filter By Application (Aka Povider) And Check Output
     Filter By Using More Than One Filter And Check Output
 
+Verify App Name On Resource Tile
+    [Documentation]    Check that each resource tile specifies which application it refers to
+    [Tags]    Sanity
+    ...       ODS-395
+    ...       Tier1    xxxx
```
Comment (jgarciao) [inline ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot:69]: xxxx should be removed

File: ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot [hunk 2]
```diff
@@ -235,3 +244,14 @@ Set Expected Items Based On RHODS Type    # robocop: disable
...
+Validate App Name Is Present On Each Tile
+    [Documentation]    Check that each Resource tile contains
+    ...    the name of the application
+    ${elements} =    Get WebElements    //article[contains(@class, 'pf-c-card')]//div[@class="pf-c-content"]//small
+    ${len}=    Get Length    ${elements}
+    FOR    ${counter}    IN RANGE    ${len}
+        ${name}=    Get Text    ${elements}[${counter}]
+        should contain    ${name}    by
+        ${length} =	Get Length    ${name}
+        Should Be True	${length} > 5
```
Comment (jgarciao) [inline ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot:256]: You can also use `Should Start With` to verify that the app name  starts with _by_ and also use [Remove String](https://robotframework.org/robotframework/latest/libraries/String.html#Remove%20String) to get it removed from the  string and obtain the app name

**Comment by nehachopra27** [comment]
**Tests**: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1659/console

Passing

**Review by jgarciao — CHANGES_REQUESTED** [review]

**Review by jgarciao — APPROVED** [review]
lgtm

**Review by bdattoma — APPROVED** [review]

**Pull request merged**